robot_name: SO101
flask_url: 127.0.0.1
flask_port: 5000
joint_names:
- shoulder_pan
- shoulder_lift
- elbow_flex
- wrist_flex
- wrist_roll
- gripper
reset_joint_positions:
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
bounding_box:
  min_translation:
  - -0.18228775262832642
  - -0.364990234375
  - 0.0787857174873352
  max_translation:
  - 0.21236354112625122
  - -0.12738528847694397
  - 0.28806227445602417
  min_rotation:
  - -178.56728965720214
  - -30.86477759763056
  - -97.70532157667657
  max_rotation:
  - 179.84548506835847
  - 23.5042925239172
  - 15.897688498658072
